src/gen/gen_vimdoc.lua

```diff
@@ -436,6 +429,21 @@ local config = {
       return { 'vim.pack' }
     end,
   },
+  plugins = {
+    filename = 'runtime-plugins.txt',
```

Comment:
"runtime" is mostly redundant

```suggestion
    filename = 'plugins.txt',
```